Jenkinsfile

```
@@ -140,7 +140,10 @@ def useBuildCache() {
     labelNoBuildCache = pullRequest.labels.contains("build-no-cache")
   }
   def noBuildCache = (env.BRANCH_NAME == 'jetty-12.0.x') || labelNoBuildCache;
-  return !noBuildCache;
+  if(pullRequest.labels.contains("force-build-cache")) {
```

I like that build labels start with "build", so how about:
```suggestion
  if(pullRequest.labels.contains("build-use-cache")) {
```